api/openapi-spec/swagger.json

```diff
@@ -21420,6 +21420,10 @@
       },
       "x-kubernetes-list-type": "atomic"
      },
+     "hostname": {
+      "description": "Specifies the hostname of the clone vmi If not specified, the hostname will be set to the name of the source vmi.",
```

Comment:
nit: what will happen if the source VMI hostname is empty? is it worth document that empty field will be omitted?

Reply:
yes, if we don't set hostname and the vm will use the original name.